statsmodels/stats/tests/test_outliers_influence.py

```diff
@@ -14,3 +17,8 @@ def test_reset_stata():
     stat = reset_ramsey(res, degree=4)
     assert_almost_equal(stat.fvalue[0, 0], 1.52, decimal=2)
     assert_almost_equal(stat.pvalue, 0.2221, decimal=4)
+
+    exog_idx = list(data.columns).index('urban')
+    X_arr = np.asarray(data)
+    vif = variance_inflation_factor(X_arr, exog_idx)
```

Comment:
You'll want `X_arr` here to be a DataFrame instead of an array to make sure your code changes are tested.